lib/widgets/icon.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class DescriptionIcon extends StatelessWidget {
+  const DescriptionIcon({
+    super.key,
+    required this.icon,
+    required this.number,
+  });
+
+  static const Color iconColor = Colors.yellow;
+
+  final IconData icon;
+  final int number;
```

Comment:
I'd choose a more descriptive name for this variable

Reply:
solved